dashboard/test/lib/projects_list_test.rb

```diff
@@ -0,0 +1,30 @@
+require 'test_helper'
+
+class ProjectsListTest < ActionController::TestCase
+  CHANNEL_ID = 'STUB_CHANNEL_ID'
+
+  setup do
+    @student = create :student
+
+    student_project_value = '{"name":"Bobs App","level":"/projects/applab","createdAt":"2017-01-24T16:41:08.000-08:00",'\
+      '"updatedAt":"2017-01-25T17:48:12.358-08:00"}'
+    @student_project = {id: 22, value: student_project_value}
+
+    hidden_project_value = '{"name":"Hidden App","level":"/projects/playlab","createdAt":"2017-01-01T00:00:00.000-08:00",'\
```

Comment:
@aoby do you think these are a good candidate for factorygirl, or is that only appropriate for activerecord objects?